docs/reference/7-docker/ed-fi-docker-compose-architecture.mdx

```diff
@@ -0,0 +1,172 @@
+for usage instructions
+:::
+
+## Configuration Options
+
+The following configurations are supported out of the box. In the diagrams
+below, the dashed line represents the internal Docker network, and all
+rectangles represent separate containers. Cylinders represent data that should
+be stored in volumes that are mapped outside of the Docker network for
+permanency.
+
+### Sandbox Configuration
+
+A Sandbox environment is generally used to support API client developers in
+developing client applications. It is not intended to be a staging environment
+for the platform host. Includes the following web applications:
+
+* Ed-Fi Web API
+* Ed-Fi Sandbox Admin
+* SwaggerUI
+
+<Tabs>
+  <TabItem value="PostgreSQL" label="PostgreSQL" default>
+    ![Sandbox Configuration - PostgreSQL](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/Sandbox-PostgreSQL.png)
+
+    [_click to enlarge_](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/Sandbox-PostgreSQL.png)
+  </TabItem>
+  <TabItem value="SQL Server" label="SQL Server">
+    ![Sandbox Configuration - SQL Server](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/Sandbox-Mssql.png)
+
+    [_click to enlarge_](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/Sandbox-Mssql.png)
+  </TabItem>
+</Tabs>
+
+
+### SingleTenant Configuration
+
+This configuration shows a basic example of a single tenant configuration where
+a single ODS database serves all data going through the API. While this example
+has one ODS for demonstration purpose, single tenant deployments can support
+separate ODS per school years and districts. See [API Client and ODS Instance
+Configuration](https://edfi.atlassian.net/wiki/spaces/ODSAPIS3V70/pages/24559631/API+Client+and+ODS+Instance+Configuration)
+for more information.
+
+Includes the following web applications:
+
+* Ed-Fi Web API
+* Ed-FI ODS Admin Api
+
+<Tabs>
+  <TabItem value="PostgreSQL" label="PostgreSQL" default>
+    ![SingleTenant Configuration - PostgreSQL](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/SingleTenant-PostgreSql.png)
+
+    [_click to enlarge_](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/SingleTenant-PostgreSql.png)
+  </TabItem>
+  <TabItem value="SQL Server" label="SQL Server">
+    ![SingleTenant Configuration - SQL Server](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/SingleTenant-Mssql.png)
+
+    [_click to enlarge_](https://edfidocs.blob.core.windows.net/$web/img/reference/docker/SingleTenant-Mssql.png)
+  </TabItem>
+</Tabs>
+
+### SingleTenant with ODS Context Configuration
+
+This configuration shows a basic example of _explicit_ data segmentation
+strategy based on school year, i.e. school year becomes a required part of the
+API route segments. See [Context-Based Routing for Year-Specific
+ODS](https://edfi.atlassian.net/wiki/spaces/ODSAPIS3V70/pages/24559277/Context-Based+Routing+for+Year-Specific+ODS)
```

Comment:
update this link to reference/ods-api/platform-dev-guide/configuration/context-based-routing-for-year-specific-ods